treeherder/webapp/api/logslice.py

```diff
@@ -59,10 +59,6 @@ def list(self, request, project, jm):
         if start_line >= end_line:
             return Response("``end_line`` must be larger than ``start_line``", 400)
 
-        # @todo: remove once Bug 1139517 is addressed
```

Comment:
This comment (I think I put it in there) doesn't quite tell the whole story.  When new logs are ingested, they will go in with the new names.  But we will still have at least 3 months of data with the logs that have the old name.  So, in the interim, we need to handle both names.

we should still remove this set of lines.  But then add checking for both later...